applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes.cpp

```diff
@@ -157,7 +164,54 @@ void TwoFluidNavierStokes<TElementData>::CalculateLocalSystem(
                     ComputeGaussPointEnrichmentContributions(data, Vtot, Htot, Kee_tot, rhs_ee_tot);
                 }
 
-                CondenseEnrichment(data, rLeftHandSideMatrix, rRightHandSideVector, Htot, Vtot, Kee_tot, rhs_ee_tot);
+                if (rCurrentProcessInfo[SURFACE_TENSION]){
```

Comment:
I'd encapsulate this entire block in a `AddSurfaceTensionContribution` function.

Reply:
Same as below, I think it would make it difficult to add more features in the future. If there is no performance/technical issue, I prefer to keep them separate.

Reply:
Well, I don't think it makes difficult to add more features. Instead of adding them in here you add them in the auxiliary private method. By doing so we keep the `CalculateLocalSystem` cleaner.

Reply:
Done.